client/src/components/Profile/Profile.js

```diff
@@ -354,6 +354,8 @@ Profile.propTypes = {
 
 Profile.defaultProps = {
   user: null,
+  editing: null,
```

Comment:
Why is this null as opposed to false?

Reply:
Good point, probably because I wrote this at like 2:30am :')